packages/augur-ui/src/assets/styles/_colors.less

```diff
@@ -21,6 +21,7 @@
 // Other
 @color-error: #FF2727; // 9 files
 @color-alternate-error: #F3E7E7;
+@color-closeable-notice-background: #DFF1FE;
```

Comment:
maybe should go in //ui section with other background colors

Reply:
@phoebemirman  What do you mean by //ui section with other background colors?

Reply:
I think she just meant up above in this file under the "// UI" line.